src/scripts/GameController.ts

```diff
@@ -375,6 +380,13 @@ class GameController {
                         return e.preventDefault();
                     }
                     break;
+                case Settings.getSetting('hotkey.achievementsTracker').value:
+                    // Open the achievmeents tracker
+                    if (achievements.canAccess() && !$achievementsModal.data('disable-toggle')) {
+                        $('.modal').modal('hide');
+                        $achievementsModal.modal('toggle');
+                        return e.preventDefault();
+                    }
```

Comment:
```suggestion
                    }
                    break;
```
Spamming the hotkey while the modal is closing ends up opening the Oak Item modal.